README.md

```diff
@@ -1,18 +1,35 @@
-# Caliban Toolbox: Data Engineering Tools for Single Cell Analysis
+# Caliban Toolbox: Label Creation and Processing for Single Cell Analysis
 
 [![Build Status](https://travis-ci.com/vanvalenlab/caliban-toolbox.svg?branch=master)](https://travis-ci.com/vanvalenlab/caliban-toolbox)
 [![Coverage Status](https://coveralls.io/repos/github/vanvalenlab/caliban-toolbox/badge.svg?branch=master)](https://coveralls.io/github/vanvalenlab/caliban-toolbox?branch=master)
 
-DeepCell Toolbox is a collection of data engineering tools for processing, annotating, and packaging optical microscopy images. The framework enables crowdsourced annotations and creates training data for [DeepCell](https://github.com/vanvalenlab/deepcell-tf).
+Caliban Toolbox is a collection of data engineering tools for pre- and post-processing data for [Caliban](https://github.com/vanvalenlab/caliban), our data annotation tool. The Toolbox and Caliban itself work together to curate annotations for training [DeepCell](https://github.com/vanvalenlab/deepcell-tf).
 
 The process is as follows:
-![flow](./docs/flowchart.png)
 
-Read the documentation at
+1. Raw data is imported using the data loader, which allows the user to select data based on imaging platform, cell type, and marker of interest. 
 
-## Getting Started
+2. The raw data is then run through deepcell-tf to produce predicted labels
```

Comment:
Not necessarily? Even when this is an option, there will be cases where it is better to label from scratch. Perhaps change to "data can be run through"